### ISSUE TITLE
Want to expose the "registerXXX" function directly so it can be used to add additional elements after the initial configuration

### ISSUE THREAD

**Issue by threedot14** [kaizendorks/vue-autowire#22]
Use case: The components for the application is split in two different folders, and the configuration context only permits one root folder.

Solution: use the config to initialize one of the folders and attempt call `Vue.autowire.registerComponents()` on the second folder separately,

**Comment by DaniJG** [comment]
Thinking about this, a cleaner way could be updating the current interface so you can pass an array for each of the conventions.

For example, doing 
```js
Vue.use(vueAutowire, {
  components: { 
    requireContext: [
       require.context('@/components', true, /\/(?:[^.]+|(?!\.local\.vue$)|(?!\.async\.vue$))\.vue$/),
       require.context('@/another-folder', true, /\/(?:[^.]+|(?!\.local\.vue$)|(?!\.async\.vue$))\.vue$/),
    ]
  }
})
```

Might not be too hard, at the beginning of the `registerComponents` function, you could check if the received argument is an array. Something like (pseudo untested code)
```js
function registerComponents (Vue, requireContext) {
  if(isArray(requireContext)) { // isArray to be created
    return requireContext.map(c => registerComponents(c)).flat();
  }

  // rest of existing code
}
```

**Comment by threedot14** [comment]
I agree that the array based idea works. But I feel exposing the function underneath is more flexible, in case the information required is not available at the same time to form an array.

**Comment by DaniJG** [comment]
True, this will be more flexible, lets go with it!

As a side note, be careful with _in case the information required is not available at the same time to form an array._  The convention regexes you use with `require.context` will always need to be well known in advance (basically at build time), otherwise webpack wouldnt know how to build your bundles! (Which is why, I struggle to see a scenario where you couldnt define an array in your entrypoint app.js)

**Comment by threedot14** [comment]
> I struggle to see a scenario where you couldnt define an array in your entrypoint app.js

Fair point.

**Comment by DaniJG** [comment]
Published version 0.1.9 to npm
